### PR TITLE
response letter writing process

### LINKED ISSUE #72: Write response letter for reviewers
Address specific commentaries:
- [x] 1. Background in the intro #66 #70
- [x] 2. Too little citations #66 #70
- [x] 3. Too many citations (tables and figures in the discussion section) #65 
- [x] 4. Accuracy of range maps #66 #70 
- [ ] 5. GBIF data - cleaning and retrieving #67
- [ ] 5. Absence data from occurrence points (where does this comment come from, really?)
> **Warning**
> This is the most important commentary, which led to the decision of major review. It's worth reviewing the text to double-check we are not misleading the reader to understand we are creating absence data from GBIF rather than "erasing" the range where preys are not present.
- [x] 6. Lines 7-9: Please provide a short description for each example (taxonomic errors, geographic inaccuracy, or sampling biases). #65 
- [x] 7. Lines 9-11: Please cite studies that have done this before! If there are no previous studies, it is important to highlight the novelty of your study. #70 
- [x] 8. Lines 21-23: Are the cited literature examples of models that do not take ecological interactions into account? Use e.g. before. 
- [x] 9. Lines 68-70: This mismatch cannot just be a result of the overestimation of the predator’s range but it could be the result of misestimating both the predator's and prey's range. Please rephrase the sentence. #65

### PR DESCRIPTION
**Pull request by graciellehigino**
closes #72

### CODE REVIEW

File: response-letter.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  with the exception of Figure 05 (line XX), which is an example of our results
+  to clarify an important point of the discussion.
+- Lines 22-23 (now lines XX-XX): the cited literature are not examples of models
+  that do not take ecological interactions into account, but studies that have
+  demonstrated how ecological interactions might be responsible for shaping a
+  species' distribution. We've rewritten the sentence to clarify that.
+-  Lines 68-70: we have rewritten the sentence to include the consideration
+   misestimation of both species in a trophic interaction. We have also added
+   citations and examples.
+- We have addressed the grammatical suggestions throughout the text, except for
```
Comment (graciellehigino) [inline response-letter.md:55]: [It's actually Figure 1, I'll fix that]

@fmwindsor @gabrieldansereau @FrancisBanville Do we agree with that? I thought the legend got more complicated with the suggestions from the reviewer. For comparison, what we have is:
> Figure 1: (a) Spatial distribution of species richness according to the original IUCN range maps of all 32 mammal species of the Serengeti food web. (b) Proportion of mammal species remaining in each local network (i.e., each pixel) after removing all species without a path to a primary producer. (c) Proportion of mammal species remaining in each local network as a function of the number of species given by the original IUCN range maps.

What the reviewer suggests is:
>Figure 1: (a) Spatial distribution of species richness according to the original IUCN range maps of all 32 mammal species of the Serengeti food web. (b) **_The_** proportion of mammal species remaining in each local network (i.e., each pixel) after removing all species without a path to a primary producer. (c) **_The_** proportion of mammal species remaining in each local network **_in_** function of the number of species given by the original IUCN range maps.

Comment (fmwindsor) [reply response-letter.md:55]: I think we leave it as it is!! 

I don't agree that adding "the" makes it clearer, and it definitely shouldn't be "in function" as that does not make sense.

File: response-letter.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+rewritten this section to clarify our method. Rather than using
+absence data, what we did was to transform the point data into raster files, and
+then restrict our range maps to the locations where we had GBIF occurrences.
+Therefore, the pixels where there are no GBIF occurrences are not considered
+true absence of a species, but absence of **data** about the occurrence of that
+species.
+
+Finally, we have addressed all the other minor comments throughout the text as
+described below:
+- We've added citations on the Methods section to support our argument that a
```
Comment (FrancisBanville) [inline response-letter.md:41]: *in* the method section

File: response-letter.md (hunk 1 above)
Comment (FrancisBanville) [reply response-letter.md:55]: I also agree that we should leave it as it is!

File: response-letter.md (hunk 2 above)
Comment (graciellehigino) [reply response-letter.md:41]: Good catch! Thanks!

**Pull request merged**